File: src/io/github/sokrato/ctf/text.clj
(ns io.github.sokrato.ctf.text
  (:gen-class))

(def digits "0123456789")
(def lowercase-ascii "abcdefghijklmnopqrstuvwxyz")
(def uppercase-ascii "ABCDEFGHIJKLMNOPQRSTUVWXYZ")
(def alphabet (str lowercase-ascii uppercase-ascii))
(def alpha-num (str alphabet digits))
(def punk "!\"#$%&'()*+,-./:;<=>?@[\\]^_")
(def alpha-num-punk (str alpha-num punk))

File: java/io/github/sokrato/util/Text.java
package io.github.sokrato.util;

public final class Text {
    public static final String DIGITS = "0123456789";
    public static final String LOWER_ASCII = "abcdefghijklmnopqrstuvwxyz";
    public static final String UPPER_ASCII = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    public static final String ALPHABET = LOWER_ASCII + UPPER_ASCII;
    public static final String ALPHA_NUM = ALPHABET + DIGITS;
    public static final String PUNCTUATION = "!\"#$%&'()*+,-./:;<=>?@[\\]^_";
    public static final String ALPHA_NUM_PUNC = ALPHA_NUM + PUNCTUATION;
}
